frontend/src/assets/css/tailwind.css

```diff
@@ -151,7 +151,7 @@
 }
 
 .outline-title {
-  @apply text-main w-full group items-center text-xs leading-5 font-semibold focus:outline-none select-none tracking-wider;
+  @apply text-main w-full items-center text-xs leading-5 font-semibold focus:outline-none select-none tracking-wider;
```

Comment:
The group comes from https://tailwindcss.com/docs/hover-focus-and-other-states

Reply:
It raises an exception after switching to v3. I'll seek more info in tailwindcss's migration guide.

Reply:
`@apply group` is forbidden in tailwindcss via [this issue](https://github.com/tailwindlabs/tailwindcss/issues/4553) and [this PR](https://github.com/tailwindlabs/tailwindcss/pull/4666).
We need to use `class="group"` explicitly to enable group-hover modifiers.
PTAL

Reply:
TIL